fastlane/Fastfile

```diff
@@ -18,7 +18,7 @@ platform :ios do
 
   before_all do
     # Ensure used Xcode version
-    xcversion(version: "~> 13.2")
+    xcversion(version: "~> 14.3")
```

Comment:
Should this be 14.2 to keep the linting action working?

Reply:
Worth a shot